pkg/utils/resources/resources.go

```diff
@@ -43,7 +43,12 @@ func LimitsForPods(pods ...*v1.Pod) v1.ResourceList {
 
 // Merge the resources from the variadic into a single v1.ResourceList
 func Merge(resources ...v1.ResourceList) v1.ResourceList {
-	result := v1.ResourceList{}
+	if len(resources) == 0 {
+		return v1.ResourceList{}
+
```

Comment:
```suggestion
```

Reply:
![Screen Shot 2022-03-26 at 8 47 45 PM](https://user-images.githubusercontent.com/64753/160263131-5a8aa45d-e4f9-4132-bd82-52f72a7c7a78.png)

Lol, Goland was hiding that from me.